skycoin/skywire
# Refactor internal packages

Did you run `make format && make check`?
yes

Fixes #	

 Changes:	
- Refactored httpauth package
- Refactored testhelpers package
- Removed reuseclient from httpauth
- Renamed extractError
- Removed noise from internal/vpn

How to test this PR:
1. Run a visor `./skywire-visor -c config.json` 
2. Check if everything is working
